{{cookiecutter.repostory_name}}/devops/vultr_scripts/vultr-update-cloudinit.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# update cloud-init data
+# this should be used only to UPDATE the data, initialization should be done via Terraform
+# see vultr_tf/core/vultr-cloud-init.tftpl
+import subprocess
+from pathlib import Path
+
+pwd = Path(__file__).parent
+
+# cloud-init script
+# use `vultr-cli instance user-data get <instanceID>` to get existing data
+user_data = pwd / 'userdata.txt'
+assert user_data.exists()
+
+with open(pwd / 'instances_id.txt') as f:
```

Comment:
How about minimal error-handling, informing that there's another script generating those?

Reply:
that's legit point, I would really like to avoid scope creeping on things we're bleeding money on. There's 2-3 other things which should be sanitized in this script, where we do draw the border?

Reply:
You've added readme. That's enough for me!